### ISSUE TITLE
Staring from aws-amplify@6.6.1 getting UnhandledSchemeError: Reading from "node:crypto" is not handled by plugins

### ISSUE THREAD

**Issue by mixalbl4-127** [aws-amplify/amplify-js#13817]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-js/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-js/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### JavaScript Framework

React

### Amplify APIs

Authentication

### Amplify Version

v6

### Amplify Categories

auth

### Backend

None

### Environment information

<details>

```
any env
```

</details>

### Describe the bug

Staring from aws-amplify@6.6.1 getting UnhandledSchemeError: Reading from "node:crypto" is not handled by plugins
downgrade to aws-amplify@6.6.0 helps

### Expected behavior

Success build using webpack

### Reproduction steps

1. Create project with Create React App (CRA)
2. Install aws-amplify@6.6.1
3. Try to build it

### Code Snippet

```javascript
// Put your code below this line.

```

### Log output

<details>

```
// Put your logs below this line


```

</details>

**Comment by mixalbl4-127** [comment]
possible because of that: https://github.com/aws-amplify/amplify-js/pull/13775/files#diff-ebce8d365a2c55374cd3bf7cca769aebe4fa757572c42e237242f262236b13a3

**Comment by cwomack** [comment]
Related to #13819

**Comment by cwomack** [comment]
Hello, @mixalbl4-127 and thank you for creating this issue.  We're looking into this and will comment back with any updates or progress made.

**Comment by cwomack** [comment]
@mixalbl4-127, closing this issue to consolidate it into #13819.  You can follow that issue for updates on a fix, but in the mean time please use v6.6.0 until that is released.

**Comment by HuiSF** [comment]
Hi @mixalbl4-127 Could you report which Webpack version that your React app is using? You can find the version by executing `npm why webpack` at the root of your React app. Thanks!
